From 92e0c9d5c53c87960de088cd4c75195e7f236b3a Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Sun, 13 Feb 2022 19:30:30 +0100
Subject: [PATCH] chagall: light: Use pragma once

---
 include/samsung_lights.h | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/include/samsung_lights.h b/include/samsung_lights.h
index fc059d9..c74b9dd 100644
--- a/include/samsung_lights.h
+++ b/include/samsung_lights.h
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2016 The CyanogenMod Project
- * Copyright (C) 2017 The LineageOS Project
+ * Copyright (C) 2017-2022 The LineageOS Project
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -15,8 +15,7 @@
  * limitations under the License.
  */
 
-#ifndef SAMSUNG_LIGHTS_H
-#define SAMSUNG_LIGHTS_H
+#pragma once
 
 /*
  * Board specific nodes
@@ -31,5 +30,3 @@
 
 // Uncomment to enable variable button brightness
 #define VAR_BUTTON_BRIGHTNESS 1
-
-#endif  // SAMSUNG_LIGHTS_H
\ No newline at end of file
